src/result.rs

```diff
@@ -16,7 +16,7 @@ pub struct RowWrapper {
 
 #[napi]
 pub struct CqlValueWrapper {
-    internal: CqlValue,
+    pub(crate) internal: CqlValue,
 }
```

Comment:
Naming nit: it's customary in Rust to write `inner` instead of `internal` in case of wrappers.